docs/source/enable_toolkit_selection.rst

```diff
@@ -0,0 +1,134 @@
+.. _toolkit-selection:
+
+Enable Toolkit Selection
+========================
+
+One of the more complicated aspects of Enable is the way in which it selects
+which widget toolkit (Qt, Wx, etc.) to use when constructing a :class:`Window`
+instance. Further, there is also a second choice of which kiva
+:class:`GraphicsContext` implementation to use along with the selected widget
+toolkit.
+
+There are of course defaults for both toolkit and kiva backend, which might even
+depend on the packages available in the Python environment where code is
+running. For most users, these defaults are sufficient and stable over time. For
+users that have specific needs, the widget toolkit and kiva backend can be
+explicitly specified.
+
+
+ETSConfig
+---------
+
+To control toolkit selection directly from application code, one must use
+:class:`~traits.etsconfig.etsconfig.ETSConfig`. Specifically, the
```

Comment:
For some reason when I built the docs this doesn't link to https://docs.enthought.com/traits/traits_api_reference/traits.etsconfig.html?highlight=etsconfig#traits.etsconfig.etsconfig.ETSConfig 
It just shows up bolded. The Pyface link to `Toolkit` further down works as expected.

Reply:
I believe this is and the `intersphinx_mapping` given above are correct though, so it may be unrelated to the changes here

Reply:
I noticed that too. I'm not sure why it doesn't work.